docs/docsource/installation/upgrade/xdm_migrate_from_kubedb_to_bitnami.rst

```diff
@@ -0,0 +1,282 @@
+Migrating MongoDB from KubeDB to Bitnami
+========================================
+
+Once a |product| instance is up and running, you can migrate the MongoDB
+service from KubeDB to Bitnami by following these manual procedures.
+
+.. note::
+
+   In the following documentation, `artesca` is the name of the |product|
+   instance, and the migration is intended for |product| version 2.0.1 and
+   earlier to 2.x.x.
+
+   The commands are written with the |product|
+   instance running Metalk8s, and the namespace is `zenko` and the
+   instance name is `artesca`.
+
+Before Upgrade
+==============
+
+#. Make sure all resources are in a ``Running`` state.
```

Comment:
An example of command that achieves the check would be useful.

Reply:
* added commands to acquire this info.